src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,6 +4,4 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
```

Comment:
Don't begin class implementation with an empty line. According to the checklist, redundant empty lines should be removed to adhere to clean code principles.